Classes/Models/File+Filename.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  File+Filename.swift
+//  Freetime
+//
+//  Created by Weyert de Boer on 13/10/2017.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+extension File {
+
+    /// Returns the potential programming language of the file, if unsure it will return `nil`
+    var potentialLanguage: String? {
+        guard let fileExtension = self.filename.lowercased().components(separatedBy: ".").last else { return nil }
+        switch fileExtension {
+        case "swift": return "swift"
```

Comment:
Probably want to extract a list of known languages to an enum + separate file? And maybe we can already find a list like that somewhere?

Then we can `return Language(rawValue: fileExtension)` (or something similar, eg. when there are multiple extensions for one language) instead of this weird `switch`. :D